app/Kings_fighting/kings_start_position.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+from app.Kings_fighting.addition_tools import magic_upgrades
+
+
+class King:
+    # It can be useful to check correct start parameters
```

Comment:
not sure if you need this comment. Like it should be either implemented or not, or, at least, in TODO